src/core/messenger/passback.ts

```diff
@@ -205,15 +205,34 @@ const init = (register: RegisterListener): void => {
 										`Passback: ad height is ${adHeight}`,
 									);
 									void fastdom.mutate(() => {
-										const slotHeight = `${
-											adHeight + adLabelHeight
-										}px`;
+										//The 100 is to redue CLS if Teads passes back so we don't resize back to 274px for desktop if we get an mpu
+										const slotHeight =
+											getCurrentBreakpoint() === 'mobile'
+												? `${
+														adHeight + adLabelHeight
+												  }px`
+												: `${
+														adHeight +
```

Comment:
I don't think we need to use the ad height here as the number we want is not related to it (350px + label), also I think in some circumstances the passback ad can also be 350px so that would reserve 450px.

Reply:
Do you think it would be better to check `size[1]` if it is not `350` then we add that 100? or better not touch that code at all?

Reply:
Have added a suggestion below

Reply:
Thank you for the suggestion Jake.